manifests/params.pp

```diff
@@ -94,4 +94,5 @@
   $environment_variables      = {}
   $config_variables           = {}
   $config_kernel_variables    = {}
+  $ulimit                     = 4096
```

Comment:
@michaelklishin thanks for lending your expertise!

If 1024 is the default on most distributions then I'd still think that the default ought to be the same here, with the idea that puppet's default behavior should be to not change anything about the system unless told to. Is 1024 a completely insensible default?